.github/workflows/pytest.yml

```diff
@@ -0,0 +1,28 @@
+name: Run Unit Test via Pytest  
+  
+on: [push]  
+  
+jobs:  
+  build:  
+    runs-on: ubuntu-latest  
+    strategy:  
+      matrix:  
+        python-version: ["3.10"]  
+  
+    steps:  
+      - uses: actions/checkout@v3  
+      - name: Set up Python ${{ matrix.python-version }}  
+        uses: actions/setup-python@v4  
+        with:  
+          python-version: ${{ matrix.python-version }}  
+      - name: Install dependencies  
+        run: |  
+          python -m pip install --upgrade pip
+          pip install testtools
```

Comment:
if these are part of requirements.txt, can we remove the explicit pip install here?

Reply:
will remove these ones requirements.txt PR is merged